vehicle/mkvi/software/throttle/throttle.yml

```diff
@@ -0,0 +1,91 @@
+name: throttle
+subscribe:
+  # Transmit MOB is 0
+  - name: brakelight
+    mob: 1
+  - name: dashboard
+    mob: 2
+publish:
+  - name: M192_Command_Message
+  - name: throttle
+    id: 0xC
+    freq_hz: 100
+    signals:
+      - name: throttle_status
+        slice: 0 + 8
+        unit:
+          type: enum
+          values:
+            - THROTTLE_IDLE
+            - THROTTLE_RUN
+            - THROTTLE_1_OUT_OF_RANGE
+            - THROTTLE_2_OUT_OF_RANGE
+            - THROTTLE_POSITION_IMPLAUSIBILITY
+            - THROTTLE_BRAKE_PRESSED
+      - name: throttle_1_pos
+        slice: 8 + 16
+        unit:
+          type: int16_t
```

Comment:
Shouldn't this be a int8_t since the slice is only 8 bits?

Reply:
If memory serves me, the first value is the starting bit and the second is the slice length, making this a 16 bit slice.

Reply:
Oh you're right, misread that.